library/core/src/ptr/non_null.rs

```diff
@@ -622,6 +627,11 @@ impl<T: ?Sized> NonNull<T> {
     #[stable(feature = "non_null_convenience", since = "1.80.0")]
     #[rustc_const_stable(feature = "non_null_convenience", since = "1.80.0")]
     #[rustc_allow_const_fn_unstable(unchecked_neg)]
+    #[kani::requires(!self.as_ptr().is_null())]
```

Comment:
All CI failures are related of using contracts directly from the Kani crate.
```
^^^^ use of undeclared crate or module `kani`
```
The annotations should be tool agnostic, so we have a wrapper. Please replace all contracts to use the wrapper `safety::requires`. Take a look at this [pull request](https://github.com/model-checking/verify-rust-std/pull/69/files) as an example.

Reply:
Removed thanks!